tests/gold_tests/pluginTest/test_hooks/body_buffer.test.py

```diff
@@ -0,0 +1,129 @@
+'''
+Verify HTTP body buffering.
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+
+
+def int_to_hex_string(int_value):
+    '''
+    Convert the given int value to a hex string with no '0x' prefix.
+
+    >>> int_to_hex_string(0)
+    '0'
+    >>> int_to_hex_string(1)
+    '1'
+    >>> int_to_hex_string(10)
+    'r'
+    >>> int_to_hex_string(16)
+    '0'
+    >>> int_to_hex_string(17)
+    'f1'
+    '''
```

Comment:
Wha?  When I run your function I get:
```
>>> int_to_hex_string(10)
'a'
>>> int_to_hex_string(16)
'10'
>>> int_to_hex_string(17)
'11'
>>> 
```
These are the correct answers no?

Reply:
Thanks, good catch. That was me testing the test. I wanted to make sure doctest was working, but I ran into problems getting doctest to run in an autest because of the required autest environment. Do you know how to run doctest in a test.py autest?